src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -586,6 +613,24 @@ void Increment(ref long counter)
         GC.KeepAlive(listener);
     }
 
+    private static (string, string) GetGetQueryParameters(string contentSource, ClientContext clientContext)
+    {
+        StringBuilder queryString = new StringBuilder();
+        queryString.Append($"?Var{0}={contentSource}");
+        string expectedString = contentSource;
+
+        int num = clientContext.GetRandomInt(100);
+
+        for (int i = 1; i < num; i++)
+        {
+            string vari = clientContext.GetRandomSubstring(contentSource);
+            expectedString += vari;
```

Comment:
Even for a test app this is a lot of unnecessary string allocation.  A StringBuilder here would be good as well.